build/nogo_config.json

```diff
@@ -244,7 +244,7 @@
       ".*_generated\\.go$": "ignore generated code"
     },
     "only_files": {
-      "planner/core/rule_partition_eliminate.go": "planner/core/rule_partition_eliminate code",
```

Comment:
Why? You only need to add new item.

Reply:
we don't have `rule_partition_eliminate.go`